OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -58,7 +57,7 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 
 		public void ResetLocation(Actor self)
 		{
-			Location = self.Location + Info.Offset;
+			Locations = new List<CPos> { self.Location + Info.Offset };
```

Comment:
I had this use `.Clear` in an earlier iteration but that breaks things because this method is also called to set the default rallypoint when the building is build. Since `Locations` is still `null` at that point, it crashes.